### DISCUSSION TITLE
How to use the conditioning technology class?

### DISCUSSION THREAD

**Discussion by aganter** [ZEN-universe/ZEN-garden#167] [Installation & Setup]
# Adding setConditioningTechnologies
If you would like to add conditioning technologies you need to add setConditioningTechnologies as a subset of setConversionTechnologies.
```Python
analysis["subsets"]["setConversionTechnologies"]    = ["setConditioningTechnologies"] 
```
Now you can add your conditioning technologies to you system dictionary:
```Python
analysis["subsets"]["setConditioningTechnologies "]    = ["compressor1", ..., "compressorN"] 
```
# Troubleshooting
- Make sure to add the data of your conditioning technologies to your dataset. The conditioning technologies are stored together with the conversion technologies in the folder _setConversionTechnologies_.  
- The conditioning carriers are automatically added to the system. Please make sure to add the data of your conditioning carriers to your dataset. The conditioning carriers are stored together with the carriers in the folder _setCarriers_.

**Comment by jacob-mannhardt** [comment]
@aganter is this still relevant?
